manifests/c/CologneCodeCompany/XYplorer/26.20.0200/CologneCodeCompany.XYplorer.yaml

```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: CologneCodeCompany.XYplorer
+PackageVersion: 26.20.0200

```

Comment:
```suggestion
PackageVersion: 26.20.0203
```

Reply:
Same issue as above, where is .0203 coming from?